source/docs/yearly-changes/yearly-changelog.rst

```diff
@@ -117,18 +135,39 @@ New Configs
 
 We've added several new configs. A full list of available configs is available in the ``configs`` (Java, C++, Python) namespace.
 
+Improved Support for roboRIO Motion Profiles
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Additional support has been added for various feedforward terms (kS, kG, kV and KA). There is now improved integration with roboRIO motion profiling using velocity or acceleration setpoints in various closed loop controls.
+
+.. code-block:: java
+
+   var constraints = new TrapezoidProfile.Constraints(80, 160); // 80 rps, 160 rps/s
+   var goal = new TrapezoidProfile.State(200, 0); // 200 rot, 0 rps
+   var profile = new TrapezoidProfile(constraints, goal);
+
+   var setpoint = profile.calculate(0.020);
+   m_positionControl.Position = setpoint.position;
+   m_positionControl.Velocity = setpoint.velocity;
+   m_talonFX.setControl(m_positionControl);
+
+New ``SyncCANcoder`` Remote Sensor
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Added support for ``SyncCANcoder`` feedback. This allows users to synchronize the TalonFX's internal rotor sensor against the remote CANcoder, but continue to use the rotor sensor for all closed loop control. TalonFX will continue to monitor the remote CANcoder and report if its internal position differs significantly from the reported position, or if the remote CANcoder disappears from the bus. Users may want this instead of FusedCANcoder if there is risk that the sensor can fail in a way that the sensor is still reporting "good" data, but the data does not match the mechanism, such as if the entire sensor mount assembly breaks off. Users using this over FusedCANcoder will not have the backlash compensation, as the CANcoder position is not continually fused in.
+
 Miscellaneous Improvements
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 * Orchestra has been ported from v5.
 
   * Now supports multiple devices playing a single track.
+  * Now works when the robot is disabled.
+  * A new control mode ``MusicTone`` has been added and can be used for playing a specific frequency.
 
 * Remote limits have been ported from v5.
-* Additional feedforwards in various controls.
 * Support for roboRIO motion profiles using Velocity/Acceleration setpoints in Position/Velocity controls.
```

Comment:
"Support for roboRIO motion profiles" has also been moved out, so this line can be removed